### PR TITLE
Test cleanup

### PR DESCRIPTION
**Pull request by ffrank**
Some bash code simplification, also skip ruby-based tests if necessary.

### CODE REVIEW

**Comment by roidelapluie** [comment]
Please source util.sh

**Comment by ffrank** [comment]
Thanks! Done.

File: test/test-golint.sh [hunk 1]
```diff
@@ -61,8 +64,6 @@ echo "Prev. # of issues: $COUNT1"
 echo "Curr. # of issues: $COUNT"
 echo "Issue count delta is: $DELTA %"
 if [ "$DELTA" -gt "$THRESHOLD" ]; then
-	echo "Maximum threshold is: $THRESHOLD %"
-	echo '`golint` FAIL'
```
Comment (purpleidea) [inline test/test-golint.sh]: Nit: want to add this line back?

**Comment by purpleidea** [comment]
Merged in https://github.com/purpleidea/mgmt/commit/806d4660cff61227a8ddd6651cc93be7bd169c20

**Pull request closed without merging**